cli/src/commands/build.js

```diff
@@ -92,12 +92,16 @@ const handler = async ({
             }
 
             reporter.info('Generating internationalization strings...')
+
+            // extracting the po and pot files from src/ to i18n folder
             await i18n.extract({
                 input: paths.src,
                 output: paths.i18nStrings,
                 paths,
             })
-            await i18n.generate({
+
+            // generating JSON from po files in i18n to src/locales (this folder is not in source control)
```

Comment:
Is this comment still accurate?  It looks like it generates directly into the build output